srcpkgs/libteam/files/teamd.conf

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Don't they provide such an example file? Seems wrong for us to provide one.

Reply:
Good point - there are a heap of examples that I'm putting there as vsconf - I just wasn't sure if I need the vconf handling (for upgrades)

Reply:
What you want for upgrades, if anything, is a `conf_files` entry. But it isn't necessary if the package doesn't install its own file.